README.md

```diff
@@ -24,7 +24,10 @@ Where
 - 'Test name' is a filename, which is relative to the current directory. If it
   is `"*"` (note that asterisks must be quoted in YAML), the link will apply to
   all tests in the current directory and its subdirectories.
-- Test result status is a status as defined in the wpt.fyi codebase.
+- `Test result status` is an optional field that records the WPT test result, as 
+  [defined](https://github.com/web-platform-tests/wpt.fyi/blob/master/shared/statuses.go#L52) 
+  in the wpt.fyi codebase. When the WPT version or the browser version 
```

Comment:
I think we need to be explicit and say: "It is currently unused."

Reply:
https://github.com/web-platform-tests/wpt-metadata/pull/342/files